.travis.yml

```diff
@@ -1,19 +1,23 @@
 # File managed by pluginsync
-sudo: false
+sudo: true
+services:
+- docker
 language: go
 go:
-- 1.6.3
-- 1.7.3
+- 1.6.x
+- 1.7.x
 env:
   global:
   - ORG_PATH=/home/travis/gopath/src/github.com/intelsdi-x
   - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
+  - GLIDE_HOME="${HOME}/.glide"
   matrix:
   - TEST_TYPE=small
+  - TEST_TYPE=large
   - TEST_TYPE=build
 matrix:
   exclude:
-  - go: 1.6.3
+  - go: 1.6.x
```

Comment:
We should also exclude 1.6.x from the large test. We only need large test on 1.7.x